lib/twingly/livefeed/version.rb

```diff
@@ -0,0 +1,5 @@
+module Twingly
+  module LiveFeed
+    VERSION = "5.0.1"
```

Comment:
If it is one gem, I think we should only have one version. Not sure users will understand/look for "module versions".

But we could have this repo consist of two gems that we publish, one for search and one for livefeed.